examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h

```diff
@@ -31,7 +35,7 @@ class FabricSyncAddBridgeCommand : public CHIPCommand, public CommissioningDeleg
         AddArgument("device-remote-ip", &mRemoteAddr);
     }
 
-    void OnCommissioningComplete(NodeId deviceId, CHIP_ERROR err) override;
+    void OnCommissioningComplete(chip::NodeId deviceId, CHIP_ERROR err) override;
```

Comment:
nit, does this need to change?

Reply:
As convention, we don't 'using namespace ::chip;' in the .h, we need to add 'chip::'